pdsfile.py

```diff
@@ -4632,9 +4634,6 @@ def shelf_lookup(self, shelf_type='info', volname=''):
         volname         can be used to get info about a volume when the method
                         is applied to its enclosing volset.
         """
-        # we don't have shelf files for documents
-        if self.is_documents:
-            return
 
```

Comment:
This change makes sense to me. `shelf_lookup()` only gets called if `self.info_shelf_expected` is True, and that attribute is always False for documents. So this should never be executed.